### PR TITLE
Feature/rename codecept to hermiona

### PR DESCRIPTION
**Pull request by anpleenko**
## Motivation/Description of the PR
- Description of this PR, which problem it solves
- Resolves #issueId (if applicable).

Applicable helpers:

- [ ] WebDriver
- [ ] Puppeteer
- [ ] Nightmare
- [ ] REST
- [ ] FileHelper
- [ ] Appium
- [ ] Protractor
- [ ] TestCafe
- [ ] Playwright

Applicable plugins:

- [ ] allure
- [ ] autoDelay
- [ ] autoLogin
- [ ] customLocator
- [ ] pauseOnFail
- [ ] coverage
- [ ] retryFailedStep
- [ ] screenshotOnFail
- [ ] selenoid
- [ ] stepByStepReport
- [ ] stepTimeout
- [ ] wdio
- [ ] subtitles

## Type of change

- [ ] :fire: Breaking changes
- [ ] :rocket: New functionality
- [ ] :bug: Bug fix
- [ ] :clipboard: Documentation changes/updates
- [ ] :hotsprings: Hot fix
- [ ] :hammer: Markdown files fix - not related to source code
- [ ] :nail_care: Polish code

## Checklist:

- [ ] Tests have been added
- [ ] Documentation has been added (Run `npm run docs`)
- [ ] Lint checking (Run `npm run lint`)
- [ ] Local tests are passed (Run `npm test`)

### CODE REVIEW

**Comment by DavertMik** [comment]
Ahuet' 
@anpleenko are you really yobnuty?

You stole my framework and sent a pull request to this repo :facepalm: 

image

**Comment by Jka4** [comment]
if rusian didn't steal/lie/kill - he's dead

**Comment by undecimodia59** [comment]
For 0 days, the Russians did not steal anything

**Comment by fifty59nine** [comment]
For 0 days, the Russians did not steal anything

**Comment by num8er** [comment]
@DavertMik how about to write copyright violation abuse to github profile of PR author?
It's inappropriate behavior must be reported.
As far as I know MIT license allows to dispute such moments.

https://support.github.com/contact/dmca-takedown

**Comment by num8er** [comment]
@anpleenko https://anarjafarov.me/dont-worry-be-happy.mp4   ;)

File: README.md [hunk 1]
```diff
@@ -13,7 +11,7 @@ Build Status:
 [![Appium Tests](https://github.com/codeceptjs/CodeceptJS/actions/workflows/appium.yml/badge.svg)](https://github.com/codeceptjs/CodeceptJS/actions/workflows/appium.yml)
 [![TestCafe Tests](https://github.com/codeceptjs/CodeceptJS/actions/workflows/testcafe.yml/badge.svg)](https://github.com/codeceptjs/CodeceptJS/actions/workflows/testcafe.yml)
 
-# CodeceptJS [![Made in Ukraine](https://img.shields.io/badge/made_in-ukraine-ffd700.svg?labelColor=0057b7)](https://stand-with-ukraine.pp.ua)
```
Comment (num8er) [inline README.md:16]: Ruzzians never invent something.
They steal and rename.

File: README.md [hunk 2]
```diff
@@ -1,9 +1,7 @@
-[![Stand With Ukraine](https://raw.githubusercontent.com/vshymanskyy/StandWithUkraine/main/banner-direct-single.svg)](https://stand-with-ukraine.pp.ua)
```
Comment (num8er) [inline README.md:1]: @anpleenko  це фiаско братан!
вы кацапы кроме воровства ничего не умеете

**Pull request closed without merging**